src/unix/linux_like/linux/gnu/b32/m68k/align.rs

```diff
@@ -0,0 +1,7 @@
+s_no_extra_traits! {
+    #[allow(missing_debug_implementations)]
+    #[repr(align(2))]
+    pub struct max_align_t {
+        priv_: [i8; f12]
```

Comment:
What `f12` here?

Reply:
I made that change because @jrtc27 suggested it here https://twitter.com/colonia00/status/1490059057159417858?s=20&t=XN5ECDY5QUuYaXohzbcCjQ

Although I'm not sure about the correct syntax. If you can help me with the correct values, it would be appreciated.

Reply:
I made the change here because @jrtc27 suggested it here https://github.com/rust-lang/libc/pull/2414#pullrequestreview-759790831

I'm not sure whether it's correct though. If you could help me with the proper values that would be appreciated.

Reply:
I mean, why is it `f12` instead of `12`? It doesn't seem like it would compile as it is.

Reply:
Oh, that's a typo. It did compile though.

Thanks for catching this, will fix this in a minute.

Reply:
Fixed and pushed. Thank you for catching this!